source/braille.py

```diff
@@ -1211,6 +1213,20 @@ def _set_windowEndPos(self, endPos):
 		if region.focusToHardLeft:
 			# Only scroll to the start of this region.

```

Comment:
This function has some double looping due to my recent changes, so I'm planning to change the order a bit later today. I want it to loop through the regions only once.